curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9ebb7b12bca3025b0a935.md

```diff
@@ -7,9 +7,9 @@ dashedName: step-90
 
 # --description--
 
-It is time for another loop. Use a `for` loop to iterate through the `inputContainers` array.
+It is time for another loop. Use a `for...of` loop to iterate through the `inputContainers` array. For the loop's variable name call it `container`.
```

Comment:
```suggestion
It is time for another loop. Create a `for...of` loop named `container` to iterate through the `inputContainers` array.
```

Reply:
I think it would be clearer to say `with the variable container` instead of `for...of` loop named container. Because we want campers to know that the `for...of` loop has the variable called container

Reply:
i agree with you @jdwilkin4 🫡